docker-compose.yml

```
@@ -3,8 +3,8 @@ services:
   status-backend:
     restart: unless-stopped
     container_name: status-backend
-    image: statuscompliance/status-backend:latest
-    # build: ../status-backend
+    # image: statuscompliance/status-backend:latest
+    build: ../status-backend
```

```suggestion
    image: statuscompliance/status-backend:latest
```

This currently fails since we don't clone status-backend anywhere now. For avoiding future confusions, I would suggest removing the build key altogether and, when a development version of the image is needed, you just need to build the image independently using ``docker build`` with the latest tag as well. When you need to go back to the main version, you can pull again the image from the registry or do a ``docker tag`` of the old sha.

This allows you to separate properly the staging vs production and avoid further mistakes.